src/zope/testing/setupstack.txt

```diff
@@ -224,7 +224,8 @@ In addition to a tearDown method, the class provides methods:
 
 Here's an example:
 
-    >>> open('t', 'w').close()
+    >>> os.listdir('.') != []
+    True
```

Comment:
This is effectively just here to check that `MyTests` actually changes directory, right?  It could probably use a comment explaining that, since it's a bit indirect.

Reply:
AFAICT, yes.

Reply:
please help me get a little cup of care for writing that comment, because I'm fresh out!

Reply:
Maybe something like "Make sure the current directory is non-empty; ``MyTests.test`` will check that its current directory is empty, thereby proving that ``setUpDirectory()`` changes to a new temporary directory."